src/main.jsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.jsx";
+import "./index.css";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>,
-)
+  </React.StrictMode>
+);
```

Comment:
Hey Grant, it looks like you're still getting a bunch of prettier changes showing up in the PR here. How are you currently utilizing prettier in your workflow?

Reply:
I'd be happy to talk through this more in our next 1:1 session.